testing/console-sa-secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
why do you need this @cniackz ? resources/base/console-ui.yaml is already updated with this secret creation

Reply:
Oh wow, sorry about it, didn't know.
I should have grep/looked for it prior adding it.
Ok, thank you very much Harsha, I will re-use it then 👍